packages/runner-ct/src/app/app.tsx

```diff
@@ -112,14 +118,14 @@ const App: React.FC<AppProps> = observer(
                   className="menu-toggle"
                   aria-label="Open the menu"
                 >
-                  <i className="fa fa-bars" aria-hidden="true"/>
```

Comment:
Maybe we can use font awesome s for icons package? https://fontawesome.com/how-to-use/on-the-web/using-with/react

Reply:
Seems we already have fa globally (somehow). Any benefit to pulling in *another* package?